benchmarks/test_cls.py

```diff
@@ -1,11 +1,14 @@
 import numpy as np
 import pyccl as ccl
+import time
 import os
 import pytest
 
 
 @pytest.fixture(scope='module', params=['analytic', 'histo'])
 def set_up(request):
+    t0 = time.time()
+
```

Comment:
Do you intend to leave these here or were you gonna remove the timings?

Reply:
They were very useful for testing the openmp. Happy to remove them if you’d like.

Reply:
no, just checking it was intentional